frontend/src/scenes/settings/environment/sessionReplayIngestionControlLogic.ts

```diff
@@ -208,14 +218,14 @@ export const sessionReplayIngestionControlLogic = kea<sessionReplayIngestionCont
         },
     })),
     sharedListeners(({ values }) => ({
-        saveUrlTriggers: async () => {
+        saveUrlBlocklists: async () => {
```

Comment:
should we still have `saveUrlTriggers` here?

Reply:
Wait, even I'm confused what happened here, I think this is wrong